app/containers/SearchContainer/index.js

```diff
@@ -0,0 +1,100 @@
+/**
+ *
+ * SearchContainer
+ *
+ */
+
+import React, { memo, useEffect } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { injectIntl, FormattedMessage as T } from 'react-intl';
+import { createStructuredSelector } from 'reselect';
+import { compose } from 'redux';
+import { injectSaga } from 'redux-injectors';
+import { selectSearchContainer, selectTrackData, selectTrackError, selectSearchTerm } from './selectors';
+import searchContainerSaga from './saga';
+import { Input } from 'antd';
+import get from 'lodash/get';
+import { isEmpty } from 'lodash';
+import debounce from 'lodash/debounce';
+import { searchContainerCreators } from './reducer';
+
+const { Search } = Input;
+
+export function SearchContainer({
+  dispatchTrackInfo,
+  dispatchClearTrackInfo,
+  intl,
+  searchTerm,
+  trackError,
+  trackData
+}) {
+  useEffect(() => {
+    get(trackData) || trackError;
+  }, [trackData]);
+
+  useEffect(() => {
+    if (searchTerm && !trackData) {
+      dispatchTrackInfo(searchTerm);
+    }
+  });
+
+  const handleOnChange = (sTerm) => {
+    if (!isEmpty(sTerm)) {
+      dispatchTrackInfo(sTerm);
+    } else {
+      dispatchClearTrackInfo();
+    }
+  };
+  const debouncedHandleOnChange = debounce(handleOnChange, 200);
+  return (
+    <div>
+      <Search
+        data-testid="search-bar"
+        defaultValue={searchTerm}
+        type="text"
+        onChange={(evt) => debouncedHandleOnChange(evt.target.value)}
+        onSearch={(searchText) => debouncedHandleOnChange(searchText)}
+      />
+      <T id={'SearchContainer'} />
+      <div data-testid="for">Data</div>
+    </div>
+  );
+}
+
+SearchContainer.propTypes = {
+  dispatchTrackInfo: PropTypes.func,
+  dispatchClearTrackInfo: PropTypes.func,
+  intl: PropTypes.object,
+  trackError: PropTypes.string,
+  searchTerm: PropTypes.string,
+  trackData: PropTypes.shape({
```

Comment:
nice 🚀 

i like that you've defined the shape here, lets please also add the shape of the objects in the result array